src/drivers/boards/tap-v1/board_config.h

```diff
@@ -195,11 +195,19 @@ __BEGIN_DECLS
 #define GPIO_TIM3_CH4OUT	GPIO_TIM3_CH4OUT_1
 #define DIRECT_PWM_OUTPUT_CHANNELS	4
 
+#define GPIO_TIM3_CH1IN		GPIO_TIM3_CH1IN_1
+#define GPIO_TIM3_CH2IN		GPIO_TIM3_CH2IN_1
+#define GPIO_TIM3_CH3IN		GPIO_TIM3_CH3IN_1
+#define GPIO_TIM3_CH4IN		GPIO_TIM3_CH4IN_1
+#define DIRECT_INPUT_TIMER_CHANNELS  4
+
```

Comment:
Why were these added?

I do not think these needed. HW wise they do not make sense. Do I have an underlying dependency I need to fix?